app/services/images/optimizer.rb

```diff
@@ -23,6 +25,12 @@ def self.call(img_src, **kwargs)
       sign_url: true
     }.freeze
 
+    def self.cloudflare(img_src, **kwargs)
+      prefix = "https://#{ApplicationConfig["CLOUDFLARE_IMAGES_DOMAIN"]}/cdn-cgi/image"
+      img_src = extract_suffix_url(img_src) if img_src.include?(prefix) # We don't want to nest prefixes
```

Comment:
Since `extract_suffix_url` can return nil, is there a plan for dealing with that case gracefully (not bubbling out a NilClass exception to the caller)?

Though, I guess by somewhat luck, `https` is in `prefix` anyway, so the regex below *should* always match... I guess I'm mostly concerned about "it works this time, but what if someone calls this thing in some other context later" defensive development.

Probably can opt to not solve for this, but I figured I'd open the discussion anyway.